tests/webgl/test_static_webgl_library.py

```diff
@@ -0,0 +1,62 @@
+"""Create a static WebGL library and run it in the browser."""
+
+from __future__ import absolute_import, print_function
+
+import os, shutil, SimpleHTTPServer, SocketServer
+import tvm
+from tvm.contrib import emscripten, util
+import numpy as np
+
+def try_static_webgl_library():
+    curr_path = os.path.dirname(os.path.abspath(os.path.expanduser(__file__)))
+
+    # Change to lib/ which contains "libtvm_runtime.bc".
+    os.chdir(os.path.join(curr_path, "../../lib"))
+
+    # Create OpenGL module.
+    n = tvm.var("n")
+    A = tvm.placeholder((n,), name='A', dtype="float")
+    B = tvm.compute((n,), lambda *i: A[i], name="B")
+
+    s = tvm.create_schedule(B.op)
+    s[B].opengl()
+
+    target_host = "llvm -target=asmjs-unknown-emscripten -system-lib"
+    f = tvm.build(s, [A, B], name="identity", target="opengl",
+                  target_host=target_host)
+
+    tempdir = util.tempdir()
+
+    # Save temporary file "identity_static.bc".
+    path_obj = tempdir.relpath("identity_static.bc")
+    f.save(path_obj)
+
+    # Write device module content into C++ array.
+    path_cc = tempdir.relpath("devc.cc")
+    with open(path_cc, "w") as fp:
```

Comment:
can we directly change this line to 

```python
f.export_library(path_dso, emscripten.create_js, side_module=False, options=[...])
```

Reply:
I understand there is a bit gap, for example, we need to save the file as ```.bc``` instead of ```.o ```

Reply:
We can add a field to create_js

```python
create_js.object_format = "bc"
```

and in the export_library:
```python
if fcompile and hasattr(fcompile, "object_format"):
   suffix = fcompile.object_format

```